frontend/src/metabase/visualizations/lib/settings/column.js

```diff
@@ -390,6 +390,9 @@ export const NUMBER_COLUMN_SETTINGS = {
     title: t`Minimum number of decimal places`,
     widget: "number",
     variant: "form-field",
+    props: {
+      placeholder: t`Min.`,
```

Comment:
This is more like a product question. I [made a comment on Notion](https://www.notion.so/Improve-the-visualization-settings-user-experience-8e1fb8fadb3f4cd9ba79e3bad7b2847b?d=bbe5d10de21d4078a134561100ba4403#ac4b6678ea1a42c5b8b5a5b2515e67ec).